### DISCUSSION TITLE
Halloo ngentoters - Gua JINGAN BOT

### DISCUSSION THREAD

**Discussion by miqbal-22** [miqbal-22/mybotv3#2] [General]
Klo lu punya ide baru atau mau request fiture kasih tau gw yaa tod

**Comment by ghost** [comment]
![Screenshot_2021-05-24-05-54-22-654_com.termux.jpg](https://user-images.githubusercontent.com/65400918/119281859-f9411980-bc54-11eb-9691-fd73882ac25f.jpg)

**Reply by miqbal-22** [reply]
Iyyaa kak lagi errorr, tunggu update an terbaru nya yaa kak☺️
